extensions/wrapper/wrapper-common-mappers/build.gradle.kts

```diff
@@ -12,12 +12,17 @@ dependencies {
     api(libs.edc.coreSpi)
     api(libs.edc.transformCore)
     api(libs.edc.transformSpi)
+
     api(project(":extensions:wrapper:wrapper-common-api"))
     api(project(":utils:json-and-jsonld-utils"))
+
+    implementation(project(":extensions:placeholder-data-source"))
```

Comment:
this should not have to be here.

it should be enough if you just access the config key, and if it's not present you default back to the old URL.

in fact you could make a minimal interface here that would be required via DIYDI.

common mappers is not an EDC extension, but a pure java gradle module we use also from e.g. the AP, I think

Reply:
Refactored: no project reference needed anymore.

> it should be enough if you just access the config key, and if it's not present you default back to the old URL.

I injected the service, it needs to show its interface or implementation class.